gammapy/data/gti.py

```diff
@@ -101,10 +128,36 @@ def read(cls, filename, hdu="GTI"):
             Filename
         hdu : str
             hdu name. Default GTI.
+        format: str
+            Input format, currently only "gadf" is supported
         """
         filename = make_path(filename)
-        table = Table.read(filename, hdu=hdu)
-        return cls(table)
+        with fits.open(str(make_path(filename)), memmap=False) as hdulist:
```

Comment:
Why was this change needed? Everywhere else, we use the simple `Table.read`

Reply:
To call directly the `GTI.from_table_hdu()` method that actually does the gadf to gammapy format conversion.
Alternatively a staticmethod such as `gadf_table_to_gammapy(table)` could also be used.

Reply:
I guess the question is whether meta data accessed via `table.meta` or `hdu.header`. I have not noticed this before but the pattern of `.from_table_hdu()` is unusual. We typically rely on the meta data being present in `table.meta`.

There is still https://github.com/gammapy/gammapy/blob/main/gammapy/maps/axes.py, but if I remember correctly this is only there for backwards compatibility, so we could actually deprecate it. 

@registerrier I think it is worth doing a follow-up PR to change to `GTI.from_table(..., format="gadf")` and not introduce `.from_table_hdu()`...

Reply:
We could keep `.from_table_hdu()` but all it does is really:

```python3
table = Table.read(table_hdu)
return cls.from_table(table, )
```

Reply:
Not sure it's worth to keep, however it saves one line of code...